### ISSUE TITLE
Upgrade Xcode to 10.3 for all device lab macs

### ISSUE THREAD

**Issue by liyuqian** [flutter/flutter#41577]
It seems that the [iOS CPU/GPU measurement](https://github.com/flutter/flutter/pull/41234) only works on Xcode 10.3 (and 10.2). Currently, mac3-7 are using Xcode 10.1, while mac1, mac2, mac8-10 are using Xcode 10.3 (or 10.2).

Can we please upgrade all Xcode to the same version 10.3 so the test could pass on any mac? This would allow us to unblock https://github.com/flutter/flutter/issues/33899

@digiter

**Comment by digiter** [comment]
This is a duplicate of #39921
